File: My Practice Projects/PracticeFragments/app/src/main/java/com/example/farhan/practicefragments/MainActivity.java
package com.example.farhan.practicefragments;

import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
        ft.add(R.id.FirstFragmentContainer, new FirstFragment());
        ft.add(R.id.LastFragmentContainer,new LastFragment());
        ft.commit();

        Button button = (Button) findViewById(R.id.mButton);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                changeFragment(new SwitchFragment());
            }
        });
    }

    private void changeFragment(Fragment fragment){
        FragmentTransaction ft = getSupportFragmentManager().beginTransaction();
        ft.replace(R.id.MiddleLinearLayout,fragment);
        ft.commit();
    }
}

File: My Practice Projects/ListViewInFragment/app/src/main/java/example/farhan/com/listview_in_fragment/MainActivity.kt
package example.farhan.com.listview_in_fragment

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import example.farhan.com.listview_in_fragment.fragments.ListViewFragment

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        supportFragmentManager.beginTransaction()
                .replace(R.id.root_frame, ListViewFragment())
                .commit()
    }
}
